### PR TITLE
Fix Management API Tokens issues

### PR DESCRIPTION
**Pull request by tamigoodall**
Removed outdated examples
Changed from how-to to reference listing available scopes and endpoints for spas
Updated links and link text

### CODE REVIEW

**Review by erjohnson — DISMISSED** [review]

File: articles/api/management/v2/get-access-tokens-for-spas.md [hunk 1]
```diff
@@ -38,83 +38,7 @@ With a Management API Token issued for a SPA, you can access the following scope
 | `create:current_user_device_credentials` | [POST /api/v2/device-credentials](/api/management/v2#!/Device_Credentials/post_device_credentials) |
 | `delete:current_user_device_credentials` | [DELETE /api/v2/device-credentials/{id}](/api/management/v2#!/Device_Credentials/delete_device_credentials_by_id) |
 
-::: note
 The above scopes (and endpoints) are [rate limited](/policies/rate-limits#access-tokens-for-spas).
-:::
-
-## Using a Management API Token to call the Management API from a SPA
```
Comment (zamd) [inline articles/api/management/v2/get-access-tokens-for-spas.md:45]: I'm not sure **why** to remove this section if we have still documented the use of mgmt api access token in SPAs in the earlier section. 

If we removing this section, we should remove SPA reference and only recommend using mgmt api access_token on the backend.

File: articles/api/management/v2/get-access-tokens-for-spas.md [hunk 2]
```diff
@@ -1,32 +1,32 @@
...
   - management-api
   - tokens
 contentType: 
-    - how-to
+    - reference
 useCase: invoke-api
 ---
 
-# Get Management API Tokens for Single-Page Applications
+# Scopes and Endpoints for Management API Tokens
```
Comment (zamd) [inline articles/api/management/v2/get-access-tokens-for-spas.md]: The new title doesn't make sense to me either... The doc still only talks about SPAs but we have removed the SPA from title??

File: articles/api/management/v2/get-access-tokens-for-spas.md [hunk 3]
```diff
@@ -38,83 +38,7 @@ With a Management API Token issued for a SPA, you can access the following scope
 | `create:current_user_device_credentials` | [POST /api/v2/device-credentials](/api/management/v2#!/Device_Credentials/post_device_credentials) |
 | `delete:current_user_device_credentials` | [DELETE /api/v2/device-credentials/{id}](/api/management/v2#!/Device_Credentials/delete_device_credentials_by_id) |
 
-::: note
```
Comment (zamd) [inline articles/api/management/v2/get-access-tokens-for-spas.md:41]: Why remove :::note?

**Review by zamd — COMMENTED** [review]
I have left my thoughts on the changes.

File: articles/api/management/v2/get-access-tokens-for-spas.md (hunk 2 above)
Comment (tamigoodall) [reply articles/api/management/v2/get-access-tokens-for-spas.md]: Changed back to **Get Management API Tokens for Single-page Applications**

File: articles/api/management/v2/get-access-tokens-for-spas.md (hunk 3 above)
Comment (tamigoodall) [reply articles/api/management/v2/get-access-tokens-for-spas.md:41]: Added back in.

File: articles/api/management/v2/get-access-tokens-for-spas.md (hunk 1 above)
Comment (tamigoodall) [reply articles/api/management/v2/get-access-tokens-for-spas.md:45]: Added back in but removed code snippets.

**Review by jeffreylees — APPROVED** [review]

**Pull request merged**